.github/workflows/nightly-throughput-stress.yml

```
@@ -0,0 +1,192 @@
+name: Nightly Throughput Stress
+
+on:
+  schedule:
+    # Run at 3 AM PST (11:00 UTC) - offset from existing nightly
+    - cron: '00 11 * * *'
+  push:
+    branches:
+      - nightly_tps
```

Once done testing the workflow
```suggestion
      - main
```